CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.35.0-rc0 / 2022-04-06
+
+* [CHANGE] TSDB: Delete *.tmp WAL files when Prometheus starts. #10317
+* [ENHANCEMENT] promtool: Add new flag `--lint`  for the commands `check rules` and `check config`. #10435
```

Comment:
Because we changed an exit code here, we should mark it as a `CHANGE` which I believe we did last time.

Reply:
Got it, I updated it to mark it as a `CHANGE`. I'm wondering if we should highlight that the exit code change. What do you think @LeviHarrison ?

Reply:
The sole linter is enabled by default (as it had been previously), so I believe that changing the exit code it returns fits the criteria of a change `CHANGE`.

Reply:
Ah ok, that makes sense then as a `CHANGE`, thanks for the clarification!